apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,56 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      [deleteItem]="deleteItem"
+      [addNewItem]="addNewItem"
+      [getName]="getName"
+      customClass="bg-light-blue">
+      <ng-container ngProjectAs="[image]">
+        <img src="assets/img/city.png" width="200px" />
+      </ng-container>
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+  styles: [
+    `
+      :host ::ng-deep .bg-light-blue {
+        background-color: rgba(0, 0, 250, 0.1);
+      }
+    `,
+  ],
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  constructor(
+    private http: FakeHttpService,
+    private store: CityStore,
+  ) {}
+  cities: City[] = [];
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((t) => this.store.addAll(t));
+
+    this.store.cities$.subscribe((t) => (this.cities = t));
```

Comment:
subscribe two times is unnecessary, checkout [my solution](https://github.com/tomalaforge/angular-challenges/pull/576/files#diff-df43fc7d8879cb363da27682e003fc37195cecacf7f09c42e4eccb2acb68ff7c) to see how it can be solved in a different way